api/HTMLUnknownElement.json

```diff
@@ -14,10 +14,10 @@
             "version_added": "12"
           },
           "firefox": {
-            "version_added": true
+            "version_added": "1"
```

Comment:
I took another look at http://mdn-bcd-collector.appspot.com/tests/api/HTMLUnknownElement in Firefox 3 to convince myself the results can be trusted. I still think the test is good.